### PR TITLE
[ci] WIP: fix secrets

### PR DESCRIPTION
**Pull request by alvicsam**
Closes https://github.com/paritytech/ink-waterfall/issues/20

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -32,13 +32,13 @@ workflow:
     REDISCLI_AUTH:
       vault:                       cicd/gitlab/parity/REDIS_AUTH@kv
       file:                        false
-    GITHUB_TOKEN:
-      vault:                       cicd/gitlab/parity/ink/GITHUB_TOKEN@kv
+    GITHUB_PR_TOKEN:
```
Comment (cmichi) [inline .gitlab-ci.yml:35]: The variable name also appears further down in the file, can you rename it there as well?

**Comment by cmichi** [comment]
@alvicsam Do you know if other tokens might be affected as well? For example, we have a lot here in ink! and some of them are only used when e.g. our fuzz tests would fail: https://github.com/paritytech/ink/blob/master/.gitlab-ci.yml#L86-L109.

**Comment by alvicsam** [comment]
@cmichi AKAIK they should be fine, we didn't change them

**Review by cmichi — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (cmichi) [reply .gitlab-ci.yml:35]: Ah my bad, mixed it up.

**Pull request merged**